lib/import-export-visitor.js

```diff
@@ -666,17 +713,19 @@ function safeParam(param, locals) {
 }
 
 function toModuleExport(specifierMap) {
-  const exportedKeys = specifierMap && Object.keys(specifierMap);
+  const exportedKeys = specifierMap ? Object.keys(specifierMap) : [];
```

Comment:
The cost of an empty array is nil compared to an engine invalidating JITed code because of type inconsistencies.